caasp-stack.yaml

```diff
@@ -221,8 +286,42 @@ resources:
 
             suse_caasp:
               role: admin
+
+            runcmd:
+              - mkdir /etc/caasp/cpi
+              - chmod -R 0777 /etc/caasp/cpi
+            # FIXME: add domain-id?
+            # domain-id=
+            # FIXME: add tenant-id?
+            # tenant-id=
+            write_files:
+              - path: /etc/caasp/cpi/openstack.conf
+                permissions: 0777
+                content: |
+                  auth-url=$cpi_auth_url
+                  domain-name=$cpi_domain_name
+                  tenant-name=$cpi_tenant_name
+                  region=$cpi_region
+                  username=$cpi_username
+                  password=$cpi_password
+                  subnet-id=$cpi_subnet_id
+                  floating-id=$cpi_floating_id
+                  monitor-max-retries=$cpi_lb_mon_retries
+                  bs-version = $cpi_cinder_bs_version
+                  ignore-vol-az = $cpi_ignore_vol_az
```

Comment:
Please remove spaces `ignore-vol-az=$cpi_ignore_vol_az`

Reply:
It should be `ignore-volume-az=$cpi_ignore_vol_az`

Reply:
done